configure-clusters/src/main/java/com/esri/arcgismaps/sample/configureclusters/screens/MainScreen.kt

```diff
@@ -0,0 +1,398 @@
+                        modifier = Modifier
+                            .align(Alignment.BottomEnd)
+                            .padding(bottom = 36.dp, end = 24.dp),
+                        onClick = {
+                            composableScope.launch {
+                                controlsBottomSheetState.show()
+                            }
+                        },
+                    ) {
+                        Icon(Icons.Filled.Settings, contentDescription = "Show controls")
+                    }
+                }
+                if (controlsBottomSheetState.isVisible) {
+                    ClusterControlsBottomSheet(
+                        composableScope = composableScope,
+                        controlsBottomSheetState = controlsBottomSheetState,
+                        showClusterLabels = mapViewModel.showClusterLabels,
+                        updateClusterLabelState = mapViewModel::updateShowClusterLabelState,
+                        clusterRadiusOptions = mapViewModel.clusterRadiusOptions,
+                        clusterRadius = mapViewModel.clusterRadius,
+                        updateClusterRadiusState = mapViewModel::updateClusterRadiusState,
+                        clusterMaxScaleOptions = mapViewModel.clusterMaxScaleOptions,
+                        clusterMaxScale = mapViewModel.clusterMaxScale,
+                        updateClusterMaxScaleState = mapViewModel::updateClusterMaxScaleState,
+                        mapScale = mapScale
+                    )
+                }
+            }
+
+            // display a bottom sheet to show popup details
+            BottomSheet(
+                isVisible = mapViewModel.showPopUpContent,
+                bottomSheetContent = {
+                    ClusterInfoContent(
+                        popUpTitle = mapViewModel.popUpTitle,
+                        popUpInfo = mapViewModel.popUpInfo,
+                        onDismiss = { mapViewModel.updateShowPopUpContentState(false) }
+                    )
+                })
+
+        }
+    )
+}
+
+/**
+ * Composable function to display the cluster controls bottom sheet.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun ClusterControlsBottomSheet(
+    composableScope: CoroutineScope,
+    controlsBottomSheetState: SheetState,
+    showClusterLabels: Boolean,
+    updateClusterLabelState: (Boolean) -> Unit,
+    clusterRadiusOptions: List<Int>,
+    clusterRadius: Int,
+    updateClusterRadiusState: (Int) -> Unit,
+    clusterMaxScaleOptions: List<Int>,
+    clusterMaxScale: Int,
+    updateClusterMaxScaleState: (Int) -> Unit,
+    mapScale: Int,
+) {
+    ModalBottomSheet(
+        modifier = Modifier.wrapContentHeight(),
+        sheetState = controlsBottomSheetState,
+        onDismissRequest = {
+            composableScope.launch {
+                controlsBottomSheetState.hide()
+            }
+        }) {
+        Row {
+            Column(
```

Comment:
The only child of this Row is the Column, which could be removed to make the Column the parent of the `ModalBottomSheet` content.